### DISCUSSION TITLE
How to test (preview) the app?

### DISCUSSION THREAD

**Discussion by ngdangtu-vn** [sveltia/sveltia-cms#36] [Q&A] [answered]
I was playing around with the project and have some modify. Now I would to preview it, see real change. What is the best way to do so?

Right now, I'm thinking about using this project as a git submodule for a preview repo. Not sure if this is a good idea :?

**Comment by kyoshino** [comment, marked as answer]
Sorry, I don’t know why this went unanswered. You may have already solved the problem, but the best way is to [work with a local repository](https://github.com/sveltia/sveltia-cms?tab=readme-ov-file#working-with-a-local-git-repository) and preview the site with your favourite framework. Now you can also [install Sveltia CMS as an npm package](https://github.com/sveltia/sveltia-cms?tab=readme-ov-file#installing-with-npm).
